qa/tasks/cephfs/kernel_mount.py

```diff
@@ -223,6 +232,42 @@ def read_debug_file(self, filename):
                 return None
             raise
 
+    def _write_dynamic_debug_file(self, enable):
+        """
+        Write to dynamic debug control file.
+        """
+        files = ["acl.c", "addr.c", "cache.c", "caps.c", "ceph_frag.c",
+                 "dir.c", "export.c", "file.c", "inode.c", "ioctl.c",
+                 "io.c", "locks.c", "mds_client.c", "mdsmap.c", "metric.c",
+                 "quota.c", "snap.c", "super.c", "util.c", "xattr.c"]
+        for filename in files:
+            if enable:
+                fdata = f"file {filename} +p"
+            else:
+                fdata = f"file {filename} -p"
```

Comment:
Why enable this on a per-file basis? Are you looking to avoid certain files in fs/ceph? It would be easier to just write this to the control file once:

`module ceph +p`

That would also ensure that it gets enabled on new files that are added.

Note too that you also need to ensure that the module is plugged in before writing to the control file. You may want to prepend this to the pyscript that gets run on the client:

`modprobe ceph`

Reply:
> Why enable this on a per-file basis? Are you looking to avoid certain files in fs/ceph? It would be easier to just write this to the control file once:
> 
> `module ceph +p`
> 
> That would also ensure that it gets enabled on new files that are added.
> 
Yeah, I just want to avoid some files those won't help, but it seems making no sense.
I will enable the whole module.

> Note too that you also need to ensure that the module is plugged in before writing to the control file. You may want to prepend this to the pyscript that gets run on the client:
> 
> `modprobe ceph`

Sure, will fix it.

Reply:
Rather than avoiding certain files, we probably ought to fix the client to not be quite so chatty when debug logging is enabled. There are some dynamic debug messages that aren't particularly helpful and could probably be removed or toned down in some fashion.

Reply:
Yeah, sound reasonable. I also found the kernel ceph will produce more logs than the libcephfs.